runtime-modules/bounty/src/lib.rs

```diff
@@ -410,28 +416,24 @@ pub struct EntryRecord<AccountId, MemberId, BlockNumber, Balance> {
 
     /// Signifies that an entry has at least one submitted work.
     pub work_submitted: bool,
-
-    /// Optional oracle judgment for the work entry.
-    /// Absent value means neither winner nor rejected entry - "legitimate user" that gets their
-    /// stake back without slashing but doesn't get a reward.
-    pub oracle_judgment_result: Option<OracleWorkEntryJudgment<Balance>>,
 }
 
 /// Defines the oracle judgment for the work entry.
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug, Copy)]
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug)]
 pub enum OracleWorkEntryJudgment<Balance> {
     /// The work entry is selected as a winner.
     Winner { reward: Balance },
 
     /// The work entry is considered harmful. The stake will be slashed.
-    Rejected,
-}
+    Rejected {
+        ///The percent share (0 - 1) to slash.
+        slashing_share: Perbill,
 
-impl<Balance> Default for OracleWorkEntryJudgment<Balance> {
-    fn default() -> Self {
-        Self::Rejected
-    }
+        /// After slash takes place the rest of the locked balance will be unlocked,
+        /// the council has the option to give description why slash happened.
+        action_justification: Option<Vec<u8>>,
```

Comment:
You have to have the type in this case, it throws an error if you write `action_justification: Option<Vec<_>>`.
You can use _ for example in a colection, `a.iter().map(|x| ...).collect::<Vec<_>>();` but that's because the type  was already defined.

Reply:
Fixed